### PR TITLE
added layouts to the kbd6x for the tofu hhkb so new owners dont have to make it themselves

### PR DESCRIPTION
**Pull request by ChrisUC**

### CODE REVIEW

File: keyboards/kbd6x/keymaps/hhkb-default-improved/keymap.c [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  [1] = LAYOUT(
+      KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_INS, RESET,
+      KC_CAPS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP, KC_TRNS, KC_DEL,
+      KC_RCTL, KC_VOLU, KC_VOLD, KC_MUTE, KC_MPLY, KC_MSTP, KC_ASTR, KC_SLSH, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,  KC_PLUS, KC_UNDS, KC_END, KC_PGDN, KC_DOWN, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS
+      ),
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```
Comment (noroadsleft) [inline keyboards/kbd6x/keymaps/hhkb-default-improved/keymap.c]: Lines 38-55 should be deleted. The functions are deprecated, and the keymap isn't using the functionality provided.

File: keyboards/kbd6x/keymaps/hhkb-default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  [1] = LAYOUT(
+      KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_INS, KC_DEL,
+      KC_CAPS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP, KC_TRNS, RESET,
+      KC_TRNS, KC_VOLU, KC_VOLD, KC_MUTE, KC_MSTP, KC_TRNS, KC_ASTR, KC_SLSH, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,  KC_PLUS, KC_UNDS, KC_END, KC_PGDN, KC_DOWN, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS
+      ),
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```
Comment (noroadsleft) [inline keyboards/kbd6x/keymaps/hhkb-default/keymap.c]: Same as the other keymap, Lines 38-55 should be deleted.

File: keyboards/kbd6x/keymaps/hhkb-default-improved/keymap.c (hunk 1 above)
Comment (ChrisUC) [reply keyboards/kbd6x/keymaps/hhkb-default-improved/keymap.c]: those were in the default layout forgot to remove when modifying

**Comment by noroadsleft** [comment]
LGTM. :+1:

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**